src/main/kotlin/no/item/xp/plugin/GenerateTypeScriptTask.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("UNUSED_PARAMETER")
```

Comment:
I think this can be removed. See if we can use less suppress. Code that gets merged to master should be finished, therefore should not contain any unused parameters.

Reply:
removed